### PR TITLE
[DNM] Apply black for all codebase

### PR DESCRIPTION
**Pull request by menshikh-iv**
Just a demonstration of `black` code formatter, see #216 for more information:
- max line length are configurable
- `'` -> `"` configurable too

@mpenkov @piskvorky guys, do you like it?

### CODE REVIEW

**Comment by piskvorky** [comment]
That's a lot of quotation marks!

A few of the changes look questionable (like removing the brackets from `IS_PY2 = (sys.version_info[0] == 2)`), but overall I like it. I think the code looks better in this PR than before.

**Comment by mpenkov** [comment]
I think the majority of the changes are just changing single quotes to double quotes. It's a rather arbitrary thing, although it seems the community leans more on the single quote side of the argument. 

It'd be good to have a look at the results _without_ those changes to see more of the signal and less of the noise.

**Comment by menshikh-iv** [comment]
@mpenkov done, please check again (I made force push)

File: smart_open/tests/test_smart_open.py [hunk 1]
```diff
@@ -862,20 +852,7 @@ class MultistreamsBZ2Test(unittest.TestCase):
...
...\x18\x00\x01>_\x80\x00\x10@\x02\xff\xf0\x01\x07n\x00?\xe7\xff\xe00\x01\x99\xaa\x00' b'\xc0\x03F\x86\x8c#&\x83F\x9a\x03\x06\xa6\xd0\xa6\x93M\x0fQ\xa7\xa8\x06\x804hh\x12$\x11\xa4i4\xf14S' b'\xd2<Q\xb5\x0fH\xd3\xd4\xdd\xd5\x87\xbb\xf8\x94\r\x8f\xafI\x12\xe1\xc9\xf8/E\x00pu\x89\x12]\xc9' b'\xbbDL\nQ\x0e\t1\x12\xdf\xa0\xc0\x97\xac2O9\x89\x13\x94\x0e\x1c7\x0ed\x95I\x0c\xaaJ\xa4\x18L\x10' b'\x05#\x9c\xaf\xba\xbc/\x97\x8a#C\xc8\xe1\x8cW\xf9\xe2\xd0\xd6M\xa7\x8bXa<e\x84t\xcbL\xb3\xa7\xd9' b'\xcd\xd1\xcb\x84.\xaf\xb3\xab\xab\xad`n}\xa0lh\tE,\x8eZ\x15\x17VH>\x88\xe5\xcd9gd6\x0b\n\xe9\x9b' b'\xd5\x8a\x99\xf7\x08.K\x8ev\xfb\xf7xw\xbb\xdf\xa1\x92\xf1\xdd|/";\xa2\xba\x9f\xd5\xb1#A\xb6\xf6' b'\xb3o\xc9\xc5y\\\xebO\xe7\x85\x9a\xbc\xb6f8\x952\xd5\xd7"%\x89>V,\xf7\xa6z\xe2\x9f\xa3\xdf\x11' b'\x11"\xd6E)I\xa9\x13^\xca\xf3r\xd0\x03U\x922\xf26\xec\xb6\xed\x8b\xc3U\x13\x9d\xc5\x170\xa4\xfa^' b'\x92\xacDF\x8a\x97\xd6\x19\xfe\xdd\xb8\xbd\x1a\x9a\x19\xa3\x80ankR\x8b\xe5\xd83]\xa9\xc6\x08' b'\x82f\xf6\xb9"6l$\xb8j@\xc0\x8a\xb0l1..\xbak\x83ls\x15\xbc\xf4\xc1\x13\xbe\xf8E\xb8\x9d\r\xa8\x9dk' b'\x84\xd3n\xfa\xacQ\x07\xb1%y\xaav\xb4\x08\xe0z\x1b\x16\xf5\x04\xe9\xcc\xb9\x08z\x1en7.G\xfc]\xc9\x14' b'\xe1B@\xbb!8`'
```
Comment (mpenkov) [inline smart_open/tests/test_smart_open.py]: Not so sure about this

Comment (menshikh-iv) [reply smart_open/tests/test_smart_open.py]: I agree, looks strange

**Comment by mpenkov** [comment]
In general, I think this is solving a problem that smart_open doesn't have. I can't remember the last time we had an argument about code style as part of this project.

On the other hand, if you're having code style argument issues in gensim, and want to trial out this solution on smart_open, then it makes sense.

**Pull request closed without merging**